app/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomPresenter.kt

```diff
@@ -468,49 +467,81 @@ class ChatRoomPresenter @Inject constructor(
 
     fun uploadImage(roomId: String, mimeType: String, uri: Uri, bitmap: Bitmap, msg: String) {
         launchUI(strategy) {
-            view.showLoading()
+            val fileName = uriInteractor.getFileName(uri) ?: uri.toString()
+            val id = UUID.randomUUID().toString()
             try {
                 withContext(Dispatchers.Default) {
-                    val fileName = uriInteractor.getFileName(uri) ?: uri.toString()
                     if (fileName.isEmpty()) {
                         view.showInvalidFileMessage()
                     } else {
                         val byteArray =
                             bitmap.getByteArray(mimeType, 60, settings.uploadMaxFileSize())
-                        retryIO("uploadFile($roomId, $fileName, $mimeType") {
-                            client.uploadFile(
-                                roomId,
-                                fileName,
-                                mimeType,
-                                msg,
-                                description = fileName
-                            ) {
-                                byteArray.inputStream()
-                            }
-                        }
+                        val username = userHelper.username()
+                        val user = userHelper.user()
+                        val attachment = Attachment(type="file", titleLink=uri.toString(), titleLinkDownload = true, imageUrl = uri.toString())
```

Comment:
@kb0304 When uploading an image, the app offers a "Description" dialog the user can fill in, which is really the message I think, and currently it shows up with the image, which is nice.  This PR strips that message off the upload.   You can see how it currently works in the latest app version from the appStore.... even though we have removed the filename fields, we still get the msg added by the user in that "Description" dialog.  Can you add that back?

Reply:
Fixed, it was not implemented for dummy message.